lms/djangoapps/teams/plugins.py

```diff
@@ -64,18 +66,37 @@ def is_available(cls, course_key: CourseKey) -> bool:
 
     @classmethod
     def is_enabled(cls, course_key: CourseKey) -> bool:
+        """
+        Returns the enabled status of teams.
+
+        Args:
+            course_key (CourseKey): The course for which to fetch the status of teams
+        """
         return CourseOverview.get_from_id(course_key).teams_enabled
```

Comment:
Last mint question: 
`is_enabled` method will stay the same and won't change to cater the change in the PR? 
i.e. you are setting enable/disable flag on`course.teams_configuration.is_enabled = enabled` but you are fetching status from `course.teams_enabled` flag.

Reply:
e.g. `return course.teams_configuration.is_enabled`

Reply:
Yes, it can stay the same. `teams_enabled` itself checks that exact value.

Reply:
Got it, thanks

Reply:
Got it, thanks